tests/deprecated/dashboard_per_fund.py

```diff
@@ -0,0 +1,196 @@
+# @pytest.mark.mock_parameters(
```

Comment:
Will the dashboard change affect the E2E tests ?

Reply:
ah yes, forgot they dont run until test. I suspect they will break in that case. Ill investigate

Reply:
all assessment e2es passing locally